stellargraph/mapper/full_batch_generators.py

```diff
@@ -503,3 +504,30 @@ def flow(self, node_ids, targets=None):
         return RelationalFullBatchNodeSequence(
             self.features, self.As, self.use_sparse, targets, node_indices
         )
+
+
+def corrupt_full_batch_node_generator(generator):
```

Comment:
Why is this a method and not a class? You can just subclass `FullBatchNodeGenerator` and override its methods as necessary. This seems inconsistent with the rest of the library.

Reply:
I replaced this with a class in a commit pushed just before your comments - but I think this mustve been after you started the review process!